src/smt/smt_engine.cpp

```diff
@@ -946,6 +959,34 @@ class SmtEnginePrivate : public NodeManagerListener {
   std::ostream* getReplayLog() const {
     return d_managedReplayLog.getReplayLog();
   }
+  
+  //------------------------------- expression names
+  // implements setExpressionName, as described in smt_engine.h
+  void setExpressionName(Expr e, std::string name) {
+    d_exprNames[Node::fromExpr(e)] = name;
+  }
+
+  // implements getExpressionNames, as described in smt_engine.h
+  std::map< Expr, std::string > getExpressionNames() const {
```

Comment:
Nit: Maybe return an `unordered_map`? The order has no significance here.

Reply:
I'm removing this method, as it is only guessing what might be useful in the future, let's not worry about it.